BP/common/landed_titles/BP_e_spain.txt

```diff
@@ -145,35 +145,6 @@ e_spain = {
 					basque = cn_salmantida
```

Comment:
A lot of shuffling around in this file, I'm not sure I get why?

Reply:
It appears (from my testing at least) that the duchy capital bulidings seem to go to the first county listed under the duchy and the county capital is the first barony under the county. So that shuffling is just repositioning baronies/counties to ensure proper capitals.